docs/configGuide/YAML-Config-Syntax.md

```diff
@@ -557,6 +557,50 @@ We can observe that:
 * Additionally, if we set `blackouts` to `[-20]`, the arena will blackout every 20 frames (because we placed the '-' indicating repeat).
 * The blackout has no effect on any other aspect of the agent or the arena. For example, the agent will still be able to move around the arena, and the objects in the arena will still be visible to the agent. _RayCasting_ will still work. 
 
+#### EXAMPLE 8 - Multi-Arena Episodes
+
+```YAML
+!ArenaConfig
+arenas:
+  0: !Arena
+    t: 50
```

Comment:
note that "t" and "pass_mark" will be renamed to "timeLimit" and "passMark" respectively. This is for all AAI builds from v.4.1.0 and up. 

For the sake of brevity, could you replace these to their newer names?